tests/test_mergesort.py

```diff
@@ -7,13 +7,21 @@
 
 
 def test_happy_flow():
-    c = merge_sort([1,2,3], [2,3,4,9], 3, 4)
+    c = merge_sort([1,2,3], [2,3,4,9])
     assert c==[1,2,2,3,3,4,9]
 
 def test_happy_flow2():
```

Comment:
no need to have two 'happy flows' - there's an infinite number of them, can't check all; focus on the corner cases

!!! tests are documentation, they explain the 'contract' - what you wrote in the docstring